back-end/migration.js

```diff
@@ -1,4 +1,4 @@
-require('dotenv').config()
+require('dotenv').config({ path: `${__dirname}/.env` })
```

Comment:
@amelieykw I think that's the default:
```suggestion
require('dotenv').config()
```

Reply:
done

Reply:
@adamgyulavari